### PR TITLE
Updates the README to include the Data Curator Onboarding Guide, the …

### PR DESCRIPTION
**Pull request by PCWCFA**
…fork-and-pull model, and other details

# Description

Updates the README to reference the MetricsDAO Data Curators Onboarding Guide, adds a fork and pull method (as opposed to being added as a contributor to the main repository), and updates the terminal and directory where some of the commands need to be executed.

# Tests

- [NA ] Please provide evidence of your successful `dbt run` / `dbt test` here
- [NA] Any comparison between `prod` and `dev` for any schema change

# Checklist
- [ NA] Follow [dbt style guide](https://github.com/dbt-labs/corp/blob/main/dbt_style_guide.md)
- [x] Tag the person(s) responsible for reviewing proposed changes
- [NA] Notes to deployment, if a `full-refresh` is needed for any table
- [x] Run `git merge main` to pull any changes from remote into your branch prior to merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,36 +5,50 @@ Curated SQL Views and Metrics for the Near Blockchain.
...
 
-Run the follow commands from inside the Near directory (**you must complete the Getting Started steps above^^**)
+Run the follow commands from inside the Near directory (**you must complete the Setup steps above^^**)
 
 ### DBT Environment
 
-`make dbt-console`
-This will mount your local near directory into a dbt console where dbt is installed.
+1. In VSCode's terminal, type `cd near_dbt`.
+2. Then run `make dbt-console`. This will mount your local near directory into a dbt console where dbt is installed.
```
Comment (D0R11) [inline README.md:45]: Hello! Maybe you can indicate here that if you're in Windows, the terminal has to be in WSL, or else the `make` command won't run in PowerShell or command prompt?

File: README.md [hunk 2]
```diff
@@ -5,36 +5,50 @@ Curated SQL Views and Metrics for the Near Blockchain.
...
+    - You can verify that the above command ran successfull by looking at the terminal prompt. It should have changed from your Linux bash prompt to something like root@3527b594aaf0:/near#. Alternatively, you can see in the Docker Desktop app that an instance of near_dbt is now running.
```
Comment (D0R11) [inline README.md:46]: Successfully instead of successfull

File: README.md (hunk 1 above)
Comment (PCWCFA) [reply README.md:59]: Updated with 
 * WSL
 * VSCode-to-WSL instructions
 * Spelling corrections

File: README.md (hunk 2 above)
Comment (PCWCFA) [reply README.md]: Corrected.

**Review by D0R11 — APPROVED** [review]
LGTM

**Pull request merged**